frontend/src/pages/DegreeWizard/DegreeWizard.tsx

```diff
@@ -105,7 +109,12 @@ const DegreeWizard = () => {
           onCancel={() => navigate('/course-selector')}
           onOk={() => resetDegree.mutate()}
         />
-        <Title className="text">Welcome to Circles!</Title>
+        <S.HeaderWrapper>
+          <Title className="text">Welcome to Circles!</Title>
+          <Button type="primary" onClick={handleLogout}>
```

Comment:
```suggestion
          <Button danger onClick={handleLogout}>
```

This button may be a better style?